twofactor_privacyidea/js/settings-admin.js

```diff
@@ -123,40 +123,53 @@ $(document).ready(function () {
         displayServerCredentials(value);
     });
 
-    $("#piSettings #triggerchallenges").change(function() {
+    $("#piSettings #triggerchallenges").change(function () {
         var checked = $(this).is(":checked");
         setValue("triggerchallenges", checked ? "1" : "0");
         displayServerCredentials(checked);
     });
-    
+
+    /* Let the user log in if no token is found */
+    getValue("passOnNoToken", function (trigger) {
```

Comment:
Please call this #passOnNotUser, so that we get not confused in the future.
...and below.